app/public/css/styles.css

```diff
@@ -1429,3 +1429,66 @@ p {
   background-position: center;
   background-image: url('../assets/alert_icon_warning.svg');
 }
+
+/* MAIN PAGE */
+
+.page-wrapper {
+  display: flex;
+  height: 100%;
+  width: var(--width);
+  margin: 0 auto;
+}
+
+.page-main {
+  flex: 0 0 66%;
+  max-width: 66%;
+}
+
+/* CARD */
+
+.card {
+  background-color: white;
+  padding: 40px 56px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* SIDEBAR */
+
+.sidebar {
+  flex: 1;
+  margin-left: 5%;
+  padding-left: 40px;
+  padding-top: 216px;
+}
+
```

Comment:
A bottom margin between items can be achieved with the `.sidebar-item` class that will wrap each section in the sidebar.